### DISCUSSION TITLE
Drop packet mismatch

### DISCUSSION THREAD

**Discussion by WPTechnician** [Security-Onion-Solutions/securityonion#12264] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

32

### RAM

144GB

### Storage for /

200GB

### Storage for /nsm

5T

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi!  I am seeing a strange discrepancy between "Dropped Monitor Traffic" in the Grid:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/8488317f-267e-4aed-8fc3-9ae64d28be2b)

...and the "Monitor Interface Drops - Inbound":

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/1cd3b5bf-b8d6-4a44-8692-9230324432b1)

The throughput it is reporting is quite low:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/fda0f79b-47a3-42ac-aa5e-7b33eb695cc6)

I can't find anything in the documentation specific to this scenario.  This started when I installed an updated server with 2.4.  I didn't see this at all with the older server and 2.3.  No other changes have happened.  Thanks!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
What do those two graphs show when you set the time to be greater than 30 minutes? Try a 3 hour window

**Comment by WPTechnician** [comment]
Hi:

Here is the three hour window:

Traffic inbound:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/c11d037f-b0ca-459a-b861-4368ee16615a)

Monitor Interface Drops Inbound:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/19175877/aa820faf-3c0d-470a-892f-6b4967324ecc)

Thanks!
